package.json

```diff
@@ -21,10 +25,15 @@
   "author": "",
   "contributors": [],
   "dependencies": {
+    "apollo-server": "^2.10.1",
+    "apollo-server-express": "^2.10.1",
```

Comment:
Sorry @yingwang-us .  One last issue.  Why not use apollo-server-express to integrate express and apollo?  According to this https://www.apollographql.com/docs/apollo-server/v1/servers/express/ we can combine the 2 and only expose 1 port.

Reply:
@mckaymic combined the 2, the changes bellow are a little complicated, due to different auth, logging requirements between ingestion api and query api. But it works under port 3333.